src/components/image/image.tsx

```diff
@@ -112,40 +114,69 @@ export class EuiImage extends Component<EuiImageProps, State> {
     const fullScreenDisplay = (
       <EuiOverlayMask onClick={this.closeFullScreen}>
         <EuiFocusTrap clickOutsideDisables={true}>
-          <button
-            type="button"
-            className="euiImage-isFullScreen"
-            onClick={this.closeFullScreen}
-            onKeyDown={this.onKeyDown}>
-            <figure>
-              <img src={url} className="euiImage-isFullScreen__img" alt={alt} />
-              {optionalCaption}
-
-              <EuiIcon
-                type="cross"
-                color={fullScreenIconColorMap[fullScreenIconColor]}
-                className="euiImage-isFullScreen__icon"
-              />
-            </figure>
-          </button>
+          <figure
+            className="euiImage euiImage-isFullScreen"
+            role="figure"
+            aria-label={caption}>
+            <EuiI18n
+              token="euiImage.closeImage"
+              default="Close image full screen">
```

Comment:
```suggestion
              default="Close full screen image">
```

Just reads slightly better